src/App.tsx

```diff
@@ -1,73 +1,105 @@
-import React from 'react';
 import './App.scss';
+import { useState, useCallback, useEffect, useRef, useMemo } from 'react';
+import cn from 'classnames';
+import debounce from 'lodash.debounce';
 import { peopleFromServer } from './data/people';
+import { Person } from './types/Person';
+import { PeopleList } from './components/PeopleList';
+
+const DELAY = 300;
 
 export const App: React.FC = () => {
-  const { name, born, died } = peopleFromServer[0];
+  const dropRef = useRef<HTMLDivElement>(null);
+
+  const [selected, setSelected] = useState<Person | null>(null);
+  const [query, setQuery] = useState('');
+  const [appliedQuery, setAppliedQuery] = useState('');
+  const [touched, setTouched] = useState(false);
```

Comment:
```suggestion
  const [isTouched, setIsTouched] = useState(false);
```
for boolean variables
but `isTouched` is not the best name here, because we set if false once input is not focused. `isFocused` name explains it better